dlk/python/dlk/templates/Makefile.tpl

```diff
@@ -153,7 +153,7 @@ lib_x86:           CXX = g++
 lib_x86:           FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -fvisibility=hidden -pthread -g
 
 lib_aarch64:       CXX = aarch64-linux-gnu-g++
-lib_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -DUSE_NEON -fvisibility=hidden -pthread -g
+lib_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++0x -fPIC -fvisibility=hidden -pthread -g
```

Comment:
To add `CXXFLAGS +=` line is useful ?

Reply:
Useful. If we not delete -DUSE_NEON, it will not work.

Reply:
But I can't find `CXXFLAGS +=` line in `lib_aarch64`.

Reply:
I see. Sorry.
`CXXFLAGS +=`line is not meaningful.
But this PR does not care about it and for consistency it should be.

Reply:
Add `CXXFLAGS` to some cases for consistency. Thanks!